backend/catalogs/triplestore.py

```diff
@@ -22,7 +22,7 @@
   values ?r {{ {obsolete_records} }}
   ?r ?p1 ?o1;
      ?pt ?f.
-  ?f ?p2 ?o2.
+  optional {{?f ?p2 ?o2.}}
```

Comment:
This change is correct, *if* we consider it acceptable for a record to have no fields. Otherwise, this would amount to working around a data integrity issue.

Reply:
Ah, I see. A record with no fields is not very useful, but it is not incorrect either. You could think of a record for which no normalization at all has been defined, but which still has a URI, the ID of the original database and the original data.